src/Traveler.js

```diff
@@ -1,9 +1,28 @@
+// const moment = require('moment')
+import moment from 'moment';
+
+
 class Traveler {
-  constructor(id, name, travelerType) {
-    this.id = id;
-    this.name = name;
-    this.travelerType = travelerType;
+  constructor(traveler, trips) {
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+    this.allTrips = trips;
+    this.travelersTrips = [];
```

Comment:
hmm, I'm still not sure that traveler needs access to everybody else's trips.

Reply:
Yeah, I need to go back and refactor that for sure.